### PR TITLE
[TASK] CGL - Remove unused imports, use imports, add phpDoc, remove unsupported parameters

### PR DESCRIPTION
**Pull request by xf-**

### CODE REVIEW

File: Classes/Controller/BackendController.php [hunk 1]
```diff
@@ -118,7 +118,7 @@ public function buildFormAction($view = 'BuildForm') {
 	 * @return void
 	 */
 	public function buildAction($name, $author, $title, $description, $controllers, $pages, $content, $backend, $vhs, $dry, $verbose) {
-		$generator = $this->extensionService->buildProviderExtensionGenerator($name, $author, $title, $description, $controllers, $pages, $content, $backend, $vhs, $dry, $verbose);
+		$generator = $this->extensionService->buildProviderExtensionGenerator($name, $author, $title, $description, $controllers, $pages, $content, $backend, $vhs);
```
Comment (NamelessCoder) [inline Classes/Controller/BackendController.php]: Both "dry" and "verbose" are still valid arguments. Travis, git etc. arguments are removed.

File: Classes/Controller/BackendController.php [hunk 2]
```diff
@@ -166,9 +166,9 @@ public function syntaxAction(array $syntax, array $extensions, array $formats, a
 				}
 				$reportsForSyntaxName = array();
 				if ('php' === $syntaxName) {
-					$reportsForSyntaxName = $this->syntaxService->syntaxCheckPhpFilesInPath($extensionFolder . '/Classes', $filteredFiles);
+					$reportsForSyntaxName = $this->syntaxService->syntaxCheckPhpFilesInPath($extensionFolder . '/Classes');
 				} elseif ('fluid' === $syntaxName) {
-					$reportsForSyntaxName = $this->syntaxService->syntaxCheckFluidTemplateFilesInPath($extensionFolder . '/Resources', $csvFormats, $filteredFiles);
+					$reportsForSyntaxName = $this->syntaxService->syntaxCheckFluidTemplateFilesInPath($extensionFolder . '/Resources', $csvFormats);
```
Comment (NamelessCoder) [inline Classes/Controller/BackendController.php]: The filteredFiles argument should definitely still be supported. It is designed to enable users to select a sub-set of listed files using checkboxes, to only process those files. This function has worked in BE before.

File: Classes/Controller/BackendController.php (hunk 1 above)
Comment (xf-) [inline Classes/Controller/BackendController.php]: Both arguments are not supported and will be set in next 2 lines via setter.
https://github.com/FluidTYPO3/builder/pull/69/files#diff-2c8d468cf2d54911b45ca38d4787091dL122

File: Classes/Controller/BackendController.php (hunk 2 above)
Comment (xf-) [inline Classes/Controller/BackendController.php]: Only HTML files are supported and processed. My suggestion merge this and open an issue :)

**Pull request merged**